style/pages/projects-details.css

```diff
@@ -21,7 +24,68 @@
 
 .projects-details .related-projects-list .nav {
```

Comment:
Do not overuse navs and lists, technically almost everything is list, but use navs and ul/ols primarily with text or compact elements